data/beaconrestapi/src/test/java/tech/pegasys/teku/beaconrestapi/BeaconRestApiTest.java

```diff
@@ -124,6 +125,16 @@ public void RestApiShouldHaveBeaconValidatorsEndpoint() {
     verify(app).get(eq(GetValidators.ROUTE), any(GetValidators.class));
   }
 
+  @Test
+  public void ShouldHaveV1VersionEndpoint() {
```

Comment:
I just noticed these methods start with a capital

Reply:
ah oops.

Reply:
the whole file was like that, but i fixed it.